deploy/example_clusters/cr-cluster-spark245-stor.yaml

```diff
@@ -13,7 +17,9 @@
 #  jupyterhub_bind_user_dn: "cn=admin,dc=mip,dc=storage,dc=hpecorp,dc=net"
 ##  #optional parameter. "use ":" as a delimiter to provide multiple templates"
 #  jupyterhub_bind_dn_template: "cn={username},ou=People,dc=mip,dc=storage,dc=hpecorp,dc=net"
-#  jupyterhub_valid_username_regex: "^[a-z][.@a-z0-9_-]*$"
+#  jupyterhub_valid_username_regex: "^[A-Za-z][.@A-Za-z0-9_-]*$"
+##  "use ":" as a delimiter to provide multiple jupyterhub_member_of"
+#  jupyterhub_member_of: CN=Eng,OU=Engineering,DC=mip,DC=storage,DC=hpecorp,DC=net:CN=QA,OU=Engineering,DC=mip,DC=storage,DC=hpecorp,DC=net
```

Comment:
same comment as below

Reply:
we should not put our AD/LDAP info in our apps

Reply:
done. changed to some dummy values